deploy/olm-catalog/argocd-operator/0.3.0/argoproj.io_applicationsets.yaml

```diff
@@ -1,3 +1,5 @@
+
+---
 apiVersion: apiextensions.k8s.io/v1
```

Comment:
The changes to this file should be reverted. They are added by the make bundle step (not sure why, but they are in the 0.1 and 0.2 releases as well). I believe these particular changes may cause the CI tests to fail when we try to release on operator hub